Resources/doc/DSL/Contents.yml

```diff
@@ -120,7 +120,7 @@
         content_type_identifier: yyy # string|string[] a content type identifier
         creation_date:
             _operator_: value # _operator_: eq, gt, gte, lt, lte, value: unix timestamp
-        group: xxx # user id, login or email if unique
+        group: [x, y, z] # array of user group ids or remote ids
```

Comment:
q: are you sure this can only be an array and not a single id ?

Reply:
ahh, right, I was misreading the code. It can be both.